### PR TITLE
[Wire] Added crop picture feature for wire background picture

### PR DESCRIPTION
**Pull request by defnax**
* Added crop picture feature for wire background picture
* Fix smooth transformation for pixmaps
* Added stylesheet for label_masthead
* Edit profile on profile page feature
* New svg icons for wire

### CODE REVIEW

**Comment by defnax** [comment]
@csoler

**Comment by defnax** [comment]
can you check my pr @csoler

File: retroshare-gui/src/gui/TheWire/WireGroupExtra.cpp [hunk 1]
```diff
@@ -34,21 +34,26 @@ WireGroupExtra::~WireGroupExtra()
...
 
+	ui.label_masthead->setFixedSize(desired_width, desired_height);
+
+	setMasthead(QPixmap());
+}
 
 void WireGroupExtra::addMasthead()
 {
-    QPixmap img = misc::getOpenThumbnailedPicture(this, tr("Load Masthead"), 400, 100);
+    QPixmap img = misc::getOpenThumbnailedPicture(this, tr("Load Masthead"), 800, 600);
```
Comment (csoler) [inline retroshare-gui/src/gui/TheWire/WireGroupExtra.cpp:49]: absolute sizes usually don't work well. I do not know how to do better here anyway.

Comment (defnax) [reply retroshare-gui/src/gui/TheWire/WireGroupExtra.cpp:49]: i changed to this, on loading the orginal image get very blurred before crop.

**Pull request merged**